### DISCUSSION TITLE
도큐먼트 검색이안되네요? (getJoinInfo 사용법)

### DISCUSSION THREAD

**Discussion by chiiheonn** [storycraft/node-kakao#524] [General]
어느날 부터 도큐먼트에서 함수 검색이 안되는데
getJoinInfo() 사용법이 어떻게되나요?..ㅠㅠ url에서 linkid를 가져오고싶은데

[a comment by an automated account is not shown]

**Reply by kaan03** [reply]
> `CLIENT.channelList.open.getJoinInfo(link url)` 로 될 것 같습니다.

아래 사진과 같이 아무것도 안뜨면 어떻게 해야하나요?

![image](https://user-images.githubusercontent.com/85083657/127223073-9f0460b5-2d06-42fd-855a-620b94ac1471.png)

**Reply by jipsoft2** [reply]
출력하신 return 값이 promise여서 그런 듯합니다. async/await 로 해서 return값을 받으시거나 then() 으로 처리하셔야할 듯합니다.

**Reply by kaan03** [reply]
감사합니다

[a comment by an automated account is not shown]
